src/Model/Post.php

```diff
@@ -0,0 +1,390 @@
+				},
+				'commentStatus' => function () {
+					return ! empty( $this->post->comment_status ) ? $this->post->comment_status : null;
+				},
+				'pingStatus'    => function () {
+					return ! empty( $this->post->ping_status ) ? $this->post->ping_status : null;
+				},
+				'slug'          => function () {
+					return ! empty( $this->post->post_name ) ? $this->post->post_name : null;
+				},
+				'toPing'        => function () {
+					return ! empty( $this->post->to_ping ) && is_array( $this->post->to_ping ) ? implode( ',', (array) $this->post->to_ping ) : null;
+				},
+				'pinged'        => function () {
+					return ! empty( $this->post->pinged ) && is_array( $this->post->pinged ) ? implode( ',', (array) $this->post->pinged ) : null;
+				},
+				'modified'      => function () {
+					return ! empty( $this->post->post_modified ) && '0000-00-00 00:00:00' !== $this->post->post_modified ? $this->post->post_modified : null;
+				},
+				'modifiedGmt'   => function () {
+					return ! empty( $this->post->post_modified_gmt ) ? Types::prepare_date_response( $this->post->post_modified_gmt ) : null;
+				},
+				'parent'        => function () {
+					$parent_post = ! empty( $this->post->post_parent ) ? get_post( $this->post->post_parent ) : null;
+					return isset( $parent_post->ID ) && isset( $parent_post->post_type ) ? DataSource::resolve_post_object( $parent_post->ID, $parent_post->post_type ) : $parent_post;
+				},
+				'editLast'      => function () {
+					$edit_last = get_post_meta( $this->post->ID, '_edit_last', true );
+					return ! empty( $edit_last ) ? DataSource::resolve_user( absint( $edit_last ) ) : null;
+				},
+				'editLock'      => function () {
+					$edit_lock       = get_post_meta( $this->post->ID, '_edit_lock', true );
+					$edit_lock_parts = explode( ':', $edit_lock );
+					return ! empty( $edit_lock_parts ) ? $edit_lock_parts : null;
+				},
+				'enclosure'     => function () {
+					$enclosure = get_post_meta( $this->post->ID, 'enclosure', true );
+					return ! empty( $enclosure ) ? $enclosure : null;
+				},
+				'guid'          => function () {
+					return ! empty( $this->post->guid ) ? $this->post->guid : null;
+				},
+				'menuOrder'     => function () {
+					return ! empty( $this->post->menu_order ) ? absint( $this->post->menu_order ) : null;
+				},
+				'link'          => function () {
+					$link = get_permalink( $this->post->ID );
+					return ! empty( $link ) ? $link : null;
+				},
+				'uri'           => function () {
+					$uri = get_page_uri( $this->post->ID );
+					return ! empty( $uri ) ? $uri : null;
+				},
+				'commentCount'  => function () {
+					return ! empty( $this->post->comment_count ) ? absint( $this->post->comment_count ) : null;
+				},
+				'featuredImage' => function () {
+					$thumbnail_id = get_post_thumbnail_id( $this->post->ID );
+					return ! empty( $thumbnail_id ) ? DataSource::resolve_post_object( $thumbnail_id, 'attachment' ) : null;
+				},
```

Comment:
Doesn't look like we're exposing the `post_password` at the Model level. Probably something that wasn't exposed before, but we need to expose it to users with proper caps to see it.

If we had a UI in Nucleus for example where users could set a password on a Post, they'd need to be able to see the current password, if they had proper caps to see it.

Reply:
Yeah, it wasn't exposed before. I can add it in.